src/ray/protobuf/experimental/autoscaler.proto

```diff
@@ -70,9 +70,9 @@ message ResourceRequestByCount {
 message GangResourceRequest {
   // a map from bundles to the number of bundles requested.
   repeated ResourceRequest requests = 1;
-  // TODO(rickyx):
-  // Strategy of the placement group, e.g. STRICT_PACK.
-  string pg_strategy = 2;
+  // Metadata associated with the request for observability,
+  // e.g. placement group's strategy.
+  string details = 2;
```

Comment:
Does autoscaler need to have any logic based on details?

Reply:
No - the purpose is to not have autoscaler depend on this. This is mainly for `ray status` - we might remove this in the futrue if we use 2 different endpoints for autoscaler and ray status.